docs/_blog/bdk_cli_basics_00.md

```diff
@@ -0,0 +1,294 @@
+---
+title: "Command Line intro to Bitcoin Wallet Dev"
+description: "Intro to bdk-cli and wallet dev"
+authors:
+    - waterst0ne Pedro Felix
+date: "2022-09-22"
+tags: ["bdk-cli", "basics","novice"]
+permalink: "/_blog/bdk_cli_basics_00"
+---
+
+# Bicoin Dev Kit - Command Line Intro to Bitcoin Wallet Development 
+
+- My goal for this tutorial is to introduce you to the powerful command-line program named `bdk-cli` a  useuful tool for Bitcoin wallet testing. You will be exposed to many of the basic techincal skills that go into creating and managing Bitcoin wallets. 
+- If you've read most of the "Mastering Bitcoin" book, this tutorial could serve as a stepping stone into your Bitcoin wallet development journey.
+
+- This short tutorial will expose you to the [`bdk library` ](https://docs.rs/bdk/latest/bdk/) and the practical knowledge needed for Bitcoin wallet development. As a consequence you will deepen your technical understanding about Bitcoin and the bdk library.
+
+- [BDK](https://github.com/bitcoindevkit/bdk-ffi) also has language bindigs for **Kotlin/Java, Swift, Python** which enable the use of BDK's **Rust** library as an API. You can later use these similar steps to create your own Bitcoin mobile, desktop or even WebApp by using the bdk-ffi language bindings. 
+***
+
+## A few things before you begin:
+
+- Three things to look out for in each step of the tutorial:
+    - 1) :arrow_forward: - Commands for the Terminal or Shell
+    - 2) :+1: - Preview of the command output. Note, not all commands will output code. 
+    - 3) Preview Video of the tutorial for reference of what things should look like in action. 
+
+***    
+### Outline of Tutorial and Installation notes:
+    
+### Brief Outline of Tutorial
+-  Step 1: Creating a mnemonic word list + XPRV (Extended Private Key)
+-  Step 2: Generate testnet Receive Address
+-  Step 3: Send funds to newly generated address
+-  Step 4: Sync Wallet
+-  Step 5: Check Balance of Wallet
+-  Step 6: Create a Transaction (PSBT)
+-  Step 7: Sign the Transaction (PSBT)
+-  Step 8: Broadcast Transaction
+
+### To follow this tutorial you will need:
+
+-  **Linux** or **macOS** device.
+-  If on **Windows** instructions will slightly differ.
+
+***
+### Rust and Cargo installation:
+-   [Rust and Cargo Installation](https://rustup.rs/) 
+
+***
+### `bdk-cli` installation:
+-  Download the [`bdk-cli` github repository locally](https://github.com/bitcoindevkit/bdk-cli.git)
+    -  Enter the folder `cd bdk-cli`
+    -  Install `cargo install --path . --features electrum,repl,compiler `
+    -  Once installation is done exit and reopen your terminal (command-line interface)
+
+### Emoji Legend:
+
+:arrow_forward: : Commands to copied and pasted
+
+:+1: : Output/ preview of code
+
+***
+## Step 0: Check Version of bdk-cli
+
+:arrow_forward:  `bdk-cli -V`
+
+
+:+1: The output below confirms the command was successful.
+```
+bdk-cli 0.5.0
+```
+
+![](https://i.imgur.com/IcuyeMS.gif)
+
+
+ ---
+ 
```

Comment:
A preview of `bdk-cli help` and a bit of intro on each command would be helpful here..

Reply:
That makes sense in case the user wants to tinker . 💯